src/main/java/frc/robot/subsystems/drive/DriveConstants.java

```diff
@@ -45,27 +41,19 @@ public class DriveConstants {
 
   public static final double TURN_MODULE_ENCODER_GEAR_RATIO = 1.0;
 
-  public static final double MEDIUM_LINEAR_SPEED_METERS_PER_SEC = Constants.PLACEHOLDER_DOUBLE,
-      MEDIUM_LINEAR_ACCELERATION_METERS_PER_SEC_SQ = Constants.PLACEHOLDER_DOUBLE,
-      MEDIUM_ANGULAR_SPEED_RAD_PER_SEC = Constants.PLACEHOLDER_DOUBLE,
-      MEDIUM_ANGULAR_ACCELERATION_RAD_PER_SEC_SQ = Constants.PLACEHOLDER_DOUBLE;
-  public static final double SLOW_LINEAR_SPEED_METERS_PER_SEC = Constants.PLACEHOLDER_DOUBLE,
-      SLOW_LINEAR_ACCELERATION_METERS_PER_SEC_SQ = Constants.PLACEHOLDER_DOUBLE,
-      SLOW_ANGULAR_SPEED_RAD_PER_SEC = Constants.PLACEHOLDER_DOUBLE,
-      SLOW_ANGULAR_ACCELERATION_RAD_PER_SEC_SQ = Constants.PLACEHOLDER_DOUBLE;
-
   /** Distance between centers of right and left wheels on robot */
-  public static final double TRACK_WIDTH_METERS = Constants.PLACEHOLDER_DOUBLE;
+  public static final double TRACK_WIDTH_METERS = Units.inchesToMeters(21.5);
 
   /**
    * Driving Parameters - Note that these are not the maximum capable speeds of the robot, rather
-   * the allowed maximum speeds
+   * the allowed maximum speeds.
+   * Max speed is 85% free speed
    */
-  public static final double MAX_SPEED_METERS_PER_SECOND = Constants.PLACEHOLDER_DOUBLE,
+  public static final double MAX_SPEED_METERS_PER_SECOND = 0.85 * Units.feetToMeters(20.25),
```

Comment:
I'd separate out the 0.85 into a separate variable but maybe that isn't necessary

Reply:
Nah fair call, changed